doc/man3/SSL_set_async_callback.pod

```diff
@@ -0,0 +1,101 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_async_callback,
+SSL_CTX_set_async_callback_arg,
+SSL_set_async_callback,
+SSL_set_async_callback_arg,
+SSL_get_async_status,
+SSL_clear_async_status,
+SSL_async_callback_fn
+- manage asynchronous operations
+
+=head1 SYNOPSIS
+
+=for comment multiple includes
+
+ #include <openssl/ssl.h>
+
+ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
+ int SSL_CTX_set_async_callback(SSL_CTX *ctx, SSL_async_callback_fn callback);
+ int SSL_CTX_set_async_callback_arg(SSL_CTX *ctx, void *arg);
+ int SSL_set_async_callback(SSL *s, SSL_async_callback_fn callback);
+ int SSL_set_async_callback_arg(SSL *s, void *arg);
+ int SSL_get_async_status(SSL *s, int *status);
+ int SSL_clear_async_status(SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_async_callback() sets asynchronous callback function, and all SSL
+sessions generated based on this SSL_CTX can get this callback, which will be
+automatically called when engine completed the cryptography operation to notify
+the application to resume the paused work flow.
+
+SSL_CTX_set_async_callback_arg() sets the callback argument.
+
+SSL_set_async_callback() allows a SSL application to set a callback in an
+asynchronous SSL session, so that when the engine completed a cryptography
+operation, the callback can be called to notify the application to resume the
+paused work flow.
+
+SSL_set_async_callback_arg() helps to set an argument for the SSL session when
+the above callback is called.
+
+SSL_get_async_status() returns the engine status. This function facilitates the
+communication from the engine to the application. During an SSL session,
+cryptographic operation would be dispatched to the engine. The engine status is
```

Comment:
"operation would be dispatched to the engine" -> "operations are dispatched to an engine"